enhancements/extending-ldap.md

```diff
@@ -85,3 +85,31 @@ defined LDAP entity.
 ## Implementation History
 
 - 2022-07-08 Initial proposal
+
+
+### Implementation
+
+#### Superusers
+
+In addition to current superuser capabilities, this implementation will add the ability for superusers to access contents from outside its permission scope. This is a very specific use case and would only be enabled if the given FEATURE flag is set. This would allow superusers, on top of taking ownership of other namespaces, read, write, and delete contents from other namespaces, without breaking backward compatibility (i.e exising installations should be able to opt-in to get this behavior).
+
+In terms of permission, access to resources not owned by the user would be granted based on whether or not that user has the existing superuser grant (scope) or not. This means that superusers would essentially bypass standard authentication in these cases. TODO: security implications.
+
+#### Restricted Users
+
+Restricted users are a subset of Quay users whose capabilites will be a subset of current regular Quay users (excluding superusers). Some limitations we're going to impose are:
+- Inability to create organizations
+- Inability to write (Feature flagged. Should this be enforced on owner's namespace, or just enforced through quota?)
```

Comment:
I'm thinking enforced on the owner's namespace, otherwise we'd be coupling quota to security.

Reply:
This will also prevent users from writing to their own org, right?